live-build/misc/live-build-hooks/81-upgrade-repository.binary

```diff
@@ -126,6 +126,6 @@ aptly repo add upgrade-repository ./*.deb
 
 aptly publish repo -skip-signing upgrade-repository
 
-tar -cf "$APPLIANCE_VARIANT.repo" -C ~/.aptly .
+tar -czf "$APPLIANCE_VARIANT.tar.gz" -C upgrade-scripts . -C ~/.aptly .
```

Comment:
perhaps we'd want a more meaningful name than just `$APPLIANCE_VARIANT.tar.gz`? e.g. `$APPLIANCE_VARIANT-upgrade.tar.gz` or `$APPLIANCE_VARIANT.upgrade.tar.gz`

Reply:
I'm OK with whatever makes sense. Let me know what name you'd like to use, and I'll change it.

FWIW, we easily change this later, as the consumers of this image (e.g. `dx_unpack`) don't rely on the filename.

Reply:
> as the consumers of this image (e.g. dx_unpack) don't rely on the filename.

Ah, I wasn't aware of this. 
I think `.upgrade.tar.gz` would be good.